### PR TITLE
feat: UI FOR DASHBOARD (#8)

### PR DESCRIPTION
**Pull request by nileshsolanki**
feat: DASHBOARD PAGE (#8 )
https://udabuddy-nilesh.herokuapp.com/
please add this domain name for authentication in firebase sign in options

### CODE REVIEW

**Comment by littlestar642** [comment]
@nileshsolanki Added the domain in allowed domain list

File: index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const port=process.env.PORT||3000;
 app.set('view engine', 'hbs');
 app.use('/bower_components',  express.static(__dirname + '/bower_components'));
 app.use(express.static(path.join(__dirname, 'public')));
-
+app.use(express.static(path.join(__dirname, 'media')));
 
 app.get('/',(req,res)=>{
```
Comment (littlestar642) [inline index.js]: Can you please add a route to the resource.hbs in the index

Comment (nileshsolanki) [reply index.js]: sure, done

**Comment by littlestar642** [comment]
@nileshsolanki There are 13 commits in you pr. Please squash these commits by using git rebase and then once again force push

**Comment by littlestar642** [comment]
Also i saw the deployment and i rally liked that. Please squash your commits asap so that i can integrate the dashboard and carry forward further implementations into the app

**Comment by littlestar642** [comment]
@nileshsolanki Seems like you haven't rebased properly. Your changes bring along changes that are outdated now and belong to a previous commit. Please rebase again and solve your merge conflicts too.

**Comment by littlestar642** [comment]
@nileshsolanki Any ETA on this?

**Pull request closed without merging**